### PR TITLE
8904 refactor preview route

### PR DESCRIPTION
**Pull request by samMcCarthyBJSS**
Story [AB#8904](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/8904)
Task [AB#9087](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9087)

### CODE REVIEW

File: app/pages/summary/contextCreator.test.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import manifest from './manifest.json';
 import { getContext } from './contextCreator';
 import { baseUrl } from '../../config';
 
-describe('order summary preview contextCreator', () => {
+describe('order summary summary contextCreator', () => {
```
Comment (alibombaye) [inline app/pages/summary/contextCreator.test.js:5]: double summary here

File: app/pages/summary/controller.test.js [hunk 2]
```diff
@@ -7,8 +7,8 @@ jest.mock('./contextCreator', () => ({
   getContext: jest.fn(),
 }));
 
-describe('order summary preview controller', () => {
-  describe('getPreviewPageContext', () => {
+describe('order summary summary controller', () => {
```
Comment (alibombaye) [inline app/pages/summary/controller.test.js:10]: another double summary

File: app/pages/summary/ui-tests/withSavedData.ui.test.js [hunk 3]
```diff
@@ -214,7 +214,7 @@ const pageSetup = async (setup = { withAuth: true, getRoute: true }) => {
   }
 };
 
-fixture('Order Summary Preview - with saved data')
+fixture('Order Summary Summary - with saved data')
```
Comment (alibombaye) [inline app/pages/summary/ui-tests/withSavedData.ui.test.js:217]: Double summary

Comment (samMcCarthyBJSS) [reply app/pages/summary/ui-tests/withSavedData.ui.test.js:217]: auto find and replace 🙃

File: app/pages/task-list/existingorder/ui-tests/ui.test.js [hunk 4]
```diff
@@ -341,21 +341,21 @@ test('should render task 4 item 1 as text if all dependencies are not met', asyn
...
+//   ]);
+//   await pageSetup({ ...defaultPageSetup, mockData: mockOrderSummary });
+//   await t.navigateTo(pageUrl);
+
+//   const task4Item1 = Selector('li[data-test-id="task-3-item-0"]');
+
+//   await t
+//     .click(task4Item1.find('a'))
+//     .expect(getLocation()).eql(`http://localhost:1234${baseUrl}/organisation/order-id/service-recipients`);
+// });
```
Comment (alibombaye) [inline app/pages/task-list/existingorder/ui-tests/ui.test.js:358]: commented out??

Comment (alibombaye) [inline app/pages/task-list/existingorder/ui-tests/ui.test.js:358]: commented out??

Comment (samMcCarthyBJSS) [reply app/pages/task-list/existingorder/ui-tests/ui.test.js:358]: ah it's cos this doesn't run locally forgot to uncomment

File: app/helpers/api/ordapi/getOrderStatus.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  const endpoint = getOrderStatusEndpoint(orderId);
+  const orderStatusData = await getData({
+    endpoint, accessToken, logger,
+  });
+  logger.info(`Order status data returned for '${orderId}'`);
+
+  return {
+    orderStatus: orderStatusData.status,
+  };
+};
```
Comment (alibombaye) [inline app/helpers/api/ordapi/getOrderStatus.js:17]: don't think we need this

**Pull request merged**